sdcm/utils/argus.py

```diff
@@ -8,7 +9,17 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def get_argus_client(run_id: UUID | str) -> ArgusSCTClient:
+def is_uuid(uuid) -> bool:
```

Comment:
run_id can be UUID or str, for UUID it will fail. See:
```
>>> uuid.UUID(uuid.uuid4())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ls/.pyenv/versions/3.10.9/lib/python3.10/uuid.py", line 174, in __init__
    hex = hex.replace('urn:', '').replace('uuid:', '')
```

Reply:
Fixed, added special case for when UUID is passed.